### ISSUE TITLE
Create Internship Post card

### ISSUE THREAD

**Issue by jwu910** [BobaTalks/internship-tracker#25]
## Description
Create InternshipPost Card

Notes: should contain at minimum
- company name
- position title
- location
- logo
- date posted
- label[]/tag[]
- external link (apply)
- save button
- job description

Read more/collapsed card can be added after initial component creation.
Focus should be on the shape and structure of the card, and the ability to quickly reuse this component as we will loop through data to render these

![image](https://user-images.githubusercontent.com/12107969/218297132-28b3b53b-d439-40f0-852e-0a6f8e90052d.png)
https://www.figma.com/file/KFMxlfGvnZncpyvACICstf/JMT-Figma?node-id=193%3A1875&t=iZLtgwe951qhr4Ju-0

**Comment by jwu910** [comment]
I don't expect backend to have an API available at this stage, but we can create some mock JSON data to work with. I would say we still create the api methods and just return the mock data for now until we can hook this up to backend whenver that gets done.

**Comment by jwu910** [comment]
Going to break this card down into smaller tickets for the comopnent parts and we can put it all together after

**Comment by jwu910** [comment]
Blocked by #56 #57 #58 #59 

Once the above tickets are completed, this component work should begin, composing from the smaller components and using mock data input to render a sample internship post card.

**Comment by jwu910** [comment]
Once #59 is done, we should pick this up and start asap

**Comment by aqvilala** [comment]
hello I'll take this on and work on the ticket (since 3/4 of the subtickets are done) and just implement #59 whenever it is ready :D

**Comment by jwu910** [comment]
@aqvilala is there anything else you had time to work on? #59 isn't completed yet

**Comment by JennaXiao3** [comment]
> @aqvilala is there anything else you had time to work on? #59 isn't completed yet

I assigned the ticket to her to add all the components that are already done so that it would be quick to add #59 to this component when it's complete (since there are no other frontends tickets left besides the blocked tickets). What do you think of this?

**Comment by jwu910** [comment]
I guess that's fine then, but that makes this ticket still blocked by #59, lets see if there's anything else that we can shift around

**Comment by JennaXiao3** [comment]
Update: #59 is done so this is good to go!

**Comment by JennaXiao3** [comment]
I've taken on this ticket, which is handled by #157

**Comment by JennaXiao3** [comment]
this ticket has been merged so can be marked as "Done"

**Comment by JennaXiao3** [comment]
nvm still need to add save and apply

**Comment by JennaXiao3** [comment]
save and apply button has been added => now ready to be reviewed
